packages/ckeditor5-source-editing/package.json

```diff
@@ -9,6 +9,24 @@
     "ckeditor5-feature",
     "ckeditor5-plugin"
   ],
+  "main": "src/index.js",
+  "dependencies": {
+    "ckeditor5": "^27.1.0"
+  },
+  "devDependencies": {
+    "@ckeditor/ckeditor5-core": "^27.1.0",
+    "@ckeditor/ckeditor5-dev-utils": "^25.0.0",
+    "@ckeditor/ckeditor5-editor-classic": "^27.1.0",
+    "@ckeditor/ckeditor5-engine": "^27.1.0",
+    "@ckeditor/ckeditor5-essentials": "^27.1.0",
+    "@ckeditor/ckeditor5-paragraph": "^27.1.0",
+    "@ckeditor/ckeditor5-table": "^27.1.0",
+    "@ckeditor/ckeditor5-theme-lark": "^27.1.0",
+    "@ckeditor/ckeditor5-ui": "^27.1.0",
+    "@ckeditor/ckeditor5-utils": "^27.1.0",
```

Comment:
As we released the new version today (`v28.0.0`), CI prints the duplicated packages error. Please, align versions.

Reply:
Done, thanks.